client/components/language-picker/modal.jsx

```diff
@@ -417,7 +430,23 @@ export class LanguagePickerModal extends PureComponent {
 		this.handleClose();
 	};
 
-	handleClose = () => {
+	recordNewLanguagePickedEvent = ( isClosingWithoutSelection ) => {
+		const { selectedLanguageSlug } = this.state;
+
+		if (
+			isClosingWithoutSelection ||
+			this.initiallySelectedLanguageSlug === selectedLanguageSlug
+		) {
+			// we don't record an event if the language wasn't changed
+			return;
+		}
+
+		const searched = this.state.recordedTracksSearchEvent;
+		this.props.recordTracksEvent( 'calypso_language_picker_new_language_picked', { searched } );
```

Comment:
This isn't a valid event name according to our tracks event design & naming conventions, so it will end up rejected by our tracks system. See PCYsg-4S2-p2 for more details.

I'd recommend renaming it to something along those lines: 
* `calypso_language_picker_selected_language`
* `calypso_language_picker_picked_language`

Reply:
Doh! I was working off of stale information. Thank you!